backend/src/main/java/com/woowacourse/levellog/application/LevellogService.java

```diff
@@ -0,0 +1,43 @@
+package com.woowacourse.levellog.application;
+
+import com.woowacourse.levellog.domain.Levellog;
+import com.woowacourse.levellog.domain.LevellogRepository;
+import com.woowacourse.levellog.dto.LevellogCreateRequest;
+import com.woowacourse.levellog.dto.LevellogResponse;
+import javax.transaction.Transactional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class LevellogService {
+
+    private final LevellogRepository levellogRepository;
+
+    public Long save(final LevellogCreateRequest request) {
+        final Levellog levellog = new Levellog(request.getContent());
+
+        final Levellog savedLevellog = levellogRepository.save(levellog);
+        return savedLevellog.getId();
+    }
+
+    public LevellogResponse find(final Long id) {
+        final Levellog levellog = getLevellog(id);
+        return new LevellogResponse(levellog.getContent());
+    }
+
+    public void update(final Long id, final LevellogCreateRequest request) {
+        final Levellog levellog = getLevellog(id);
+        levellog.updateContent(request.getContent());
+    }
+
+    private Levellog getLevellog(final Long id) {
+        return levellogRepository.findById(id)
+                .orElseThrow();
+    }
```

Comment:
```suggestion
    private Levellog getLevellogById(final Long id) {
        return levellogRepository.findById(id)
                .orElseThrow();
    }
```
같이 명확하게 표현해주는건 어떨까요? 아니면 `getById`?

또 orElseThrow에 던져지는 예외를 적어주면 좋을 것 같아요~!

Reply:
예외가 아직 안정해져서 비워둔거라면 주석이라도 달아두면 좋지 않을까요??